gpkit/nomial_data.py

```diff
@@ -107,7 +107,7 @@ def diff(self, var):
     def __eq__(self, other):
         """Equality test"""
         if not all(hasattr(other, a) for a in ("exps", "cs", "units")):
-            return NotImplemented
+            return False
```

Comment:
Ahh I see, you mean like this: 

```python
In [8]: with SignomialsEnabled():
   ...:     s0 = Signomial(0)
   ...:

In [9]: s0
Out[9]: gpkit.Signomial(0)

In [10]: 0 == s0
Out[10]: False

In [11]: 0 != s0
Out[11]: False
```
? 

Do you believe they should both be False? I think `0 == Signomial(0)` should be True...